ui/apps/dashboard/package.json

```diff
@@ -40,7 +40,7 @@
     "@remixicon/react": "4.2.0",
     "@sentry/nextjs": "7.88.0",
     "@sindresorhus/slugify": "2.2.0",
-    "@stripe/react-stripe-js": "2.1.0",
+    "@stripe/react-stripe-js": "3.1.1",
```

Comment:
This is just the UI library, the breaking change was a component rename, and we don't use the component